var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -217,6 +217,9 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
 
     # ###################### Conflicts ##########################
 
+    conflicts("cxxstd=11", when="@develop")
+    conflicts("cxxstd=14", when="@develop")
+
```

Comment:
The existing `cxxstd` conflicts are near line 340. The `=11` is redundant with the existing requirement for `@13.2:`. We probably will be able to write the conflict as `when="@14:"` since that will include develop and be true for the next trilinos release.

Reply:
Does it look OK now? Just pushed a change.